### ISSUE TITLE
Low poly but same texture quality?!

### ISSUE THREAD

**Issue by juricagrepo** [alicevision/meshroom#1095]
Hello,

while following meshroom manual for beginners I came to the simplification and decimation part, now I'm wondering if I'm gonna follow the steps am I gonna lose texture quality after it. I would like to keep texture quality but reduce poly number. Is it possible and how to do that with meshroom or meshlab. Thank you very much!

**Comment by natowi** [comment]
As long as you do not simplify your model too much, the texture quality will be unaffected. Just create node variations to see what works best for you.

**Comment by fabiencastan** [comment]
Yes. As said by @natowi, the density of the mesh will not change the texture resolution. But if the mesh decimation change the surface position too much, it will affect the quality of the texturing in term of misalignments/blur.

**Comment by juricagrepo** [comment]
Thanks for fast response, but what should I consider as a _too much_ and how should i insert node variations in node layout?

**Comment by juricagrepo** [comment]
Thank you, folks!
